tests/functional/test_gcs_aliasing.py

```diff
@@ -0,0 +1,44 @@
+"""
+tests dedicated to the `globus gcs` aliasing of other commands,
+which specifically exercise the behavior of *aliasing*
+
+these tests do not exercise the aliased commands at all
+"""
+import pytest
+
+
+@pytest.mark.parametrize(
+    "subcommand",
+    (
+        ["collection", "delete"],
+        ["collection", "list"],
+        (
+            ["endpoint", "storage-gateway", "list"],
+            ["gcs", "storage-gateway", "list"],
+        ),
+        (
+            ["endpoint", "user-credential", "list"],
+            ["gcs", "user-credential", "list"],
+        ),
+    ),
+)
+def test_aliased_commands_have_unique_usage_lines(run_line, subcommand):
+    if isinstance(subcommand, tuple):
+        from_command = subcommand[0]
+        to_command = subcommand[1]
+    else:
+        from_command = subcommand
+        to_command = ["gcs"] + subcommand
```

Comment:
This can be simplified.

```suggestion
@pytest.mark.parametrize(
    "from_command, to_command",
    (
        ("collection delete", "gcs collection delete"),
        ("collection list", "gcs collection list"),
        ("endpoint storage-gateway list", "gcs storage-gateway list"),
        ("endpoint user-credential list", "gcs user-credential list"),
    ),
)
def test_aliased_commands_have_unique_usage_lines(run_line, from_command, to_command):
```